cartridges/app_storefront_training/cartridge/client/default/scss/global.scss

```diff
@@ -0,0 +1,4 @@
+@charset "UTF-8";
+
+@import "~base/global";
+@import "forms_framework/forms_framework.scss";
```

Comment:
Always place a line break at the end of the file.
Use "**./**" in the beginning of the path.